packages/components/src/CardGlow.astro

```diff
@@ -0,0 +1,41 @@
+---
+export interface Props {}
+---
+
+<script>
+  document.addEventListener("mousemove", (e) => {
```

Comment:
should we make it so we can apply this to anything?

Reply:
Oh interesting... you mean the dynamic glow right? I've renamed the component `DynamicGlow` and used a more generic class name as the selector so it can be applied to other elements. Actually it could even be used by arbitrary elements in theory.